clients/formdata.ts

```diff
@@ -0,0 +1,91 @@
+function escapeDots(key: string): string {
```

Comment:
I think it might worth to add some Jsdoc since these functions will be widely used